src/pages/Home/components/movie-sorting/MovieSorting.module.scss

```diff
@@ -38,12 +42,35 @@
   }
 
   .sortingList {
-    display: none;
+    position: absolute;
+    top: 100%;
+    left: 0;
+    width: 100%;
+    background: $black_color;
+    box-shadow: 0 0 2px rgba(0, 0, 0, 0.1), 0 2px 10px rgba(0, 0, 0, 0.07), 0 10px 20px rgba(0, 0, 0, 0.05), 0 10px 50px rgba(0, 0, 0, 0.05);
+    padding: 1rem 0;
+    z-index: 5;
   }
 
   .sortingItem {
+    border-bottom: 1px solid $dark_grey;
+
+    &:last-child {
+      border-bottom: 0;
+    }
   }
 
   .sortingValue {
+    display: block;
```

Comment:
If you need block behaviour use `<div>` instead of `<span>`

Reply:
changed